CHANGELOG.md

```diff
@@ -7,9 +7,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+
+- Statistic: % of Persons of each type
```

Comment:
The percentage is not actually calculated in lib-cove-bods, right?

Reply:
No, a lot of the stat's are just raw numbers. But good point, the changelog is technically wrong then